embedded/app/main/badgerloop_LV.cpp

```diff
@@ -73,7 +73,7 @@ int main() {
             data->flags->brakeSecRetr = false;
         }
 		
-        
+        csvFormatShow();
```

Comment:
This is essentially an aggressive debug print, might want to also get rid of this for the master branch